### PR TITLE
Update Django & Python support

### PR DESCRIPTION
**Pull request by hsmett**
Drop django < 1.7 support
Drop Python 2.6 support
Add test for Django 1.11

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,6 +26,7 @@ share/
 
 # Editors' temporary files
 .*.swp
+.idea
```
Comment (wo0dyn) [inline .gitignore]: Nitpick: you should add this rule to your `$ git config core.excludesfile`. 😉

Comment (hsmett) [reply .gitignore]: Thanks for the tip!

**Comment by wo0dyn** [comment]
Can you please amend the `CHANGELOG` as well, and we can merge this PR right away. 👍

**Comment by hsmett** [comment]
Done! would you like me to clean the commit history?

**Pull request merged**